source/zh-cn/docs/index.md

```diff
@@ -28,6 +28,7 @@ Hexo 是一个快速、简洁且高效的博客框架。Hexo 使用 [Markdown](h
 ### 安装 Git
 
 - Windows：下载并安装 [git](https://git-scm.com/download/win).
+  - 中国大陆地区用户：[淘宝 Git for Windows 镜像](https://npm.taobao.org/mirrors/git-for-windows/) 
```

Comment:
IMHO, a separate `blockquote` is preferred.

Reply:
> IMHO, a separate is preferred.`blockquote`

@SukkaW  Emm, Taobao only provides the git mirror of Windows. May it isn't suitable to use a blockquote. Use the list should be more appropriate. Do you have another suggestion for this?

Reply:
So why not just retain current:

```
{% note info Windows 用户 %}	
对于中国大陆地区用户，可以前往 [淘宝 Git for Windows 镜像](https://npm.taobao.org/mirrors/git-for-windows/) 下载 git 安装包。
{% endnote %}
```

Reply:
@SukkaW I had changed it.